### ISSUE TITLE
refresh text_only_repos fixture after rhbz#1608683 is resolved

### ISSUE THREAD

**Issue by ktdreyer** [red-hat-storage/errata-tool#111]
The response data for the `/text_only_repos` API endpoint will soon change, https://bugzilla.redhat.com/show_bug.cgi?id=1608683

When this does, we should refresh the static JSON fixture file at `errata_tool/tests/fixtures/errata.devel.redhat.com/api/v1/erratum/26175/te
xt_only_repos` for accuracy's sake.

**Comment by yazug** [comment]
This bug has been resolved, January of 2019

**Comment by yazug** [comment]
ktdreyer left a comment

The fixture we need to refresh is errata_tool/tests/fixtures/errata.devel.redhat.com/api/v1/erratum/33840/text_only_repos

**Comment by ktdreyer** [comment]
Yes, sorry I forgot to edit this ticket text after I merged https://github.com/red-hat-storage/errata-tool/pull/122 .
